prettify.js

```diff
@@ -1,33 +1,80 @@
 require("./prettify.less");
 
+require("./prism-config");
 var Prism = require("prismjs");
 require("prismjs/plugins/line-numbers/prism-line-numbers.js");
```

Comment:
generally w/ steal, you don't include the `.js`

Reply:
fixed